adapters/vue/RuiExpandCollapseVue/package.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "@rhythm-ui/expand-collapse-vue",
+	"version": "1.0.0",
+	"main": "lib/index.js",
+	"scripts": {
+		"start": "yarn clean && concurrently \"yarn start:babel\" \"yarn start:declarations\"",
+		"start:babel": "babel -w -s -d lib src --extensions \".ts,.vue\" --ignore *.spec.ts --copy-files",
+		"start:declarations": "tsc --declaration --emitDeclarationOnly --watch",
+		"build": "yarn build:babel && yarn build:declarations",
+		"build:babel": "babel -s -d lib src --extensions \".ts,.vue\" --ignore *.spec.ts --ignore *.d.ts --copy-files",
+		"build:declarations": "tsc --declaration --emitDeclarationOnly",
+		"clean": "rimraf lib"
+	},
+	"files": [
+		"lib/"
+	],
+	"dependencies": {
+		"@rhythm-ui/expand-collapse": "^1.0.0",
+		"vue-class-component": "^7.1.0",
+		"vue-property-decorator": "^8.2.1"
+	},
+	"devDependencies": {
+		"babel-preset-rhythm-ui-vue": "^1.0.0",
+		"vue": "^2.6.10",
```

Comment:
This looks like its already defined as a dependency of `babel-preset-rhythm-ui-vue` so we dont need it here too.